contract/src/main/java/io/neow3j/contract/Nep5.java

```diff
@@ -0,0 +1,223 @@
+package io.neow3j.contract;
+
+import io.neow3j.protocol.Neow3j;
+import io.neow3j.protocol.core.methods.response.InvocationResult;
+import io.neow3j.protocol.exceptions.ErrorResponseException;
+import io.neow3j.wallet.Account;
+
+import java.io.IOException;
+import java.math.BigInteger;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Nep5 {
+
+    private Neow3j neow3j;
+    private ScriptHash scriptHash;
+
+    private Nep5() {
+    }
+
+    private Nep5(Builder builder) {
+        this.neow3j = builder.neow3j;
+        this.scriptHash = builder.scriptHash;
+    }
+
+    /**
+     * Gets the name of the token.
+     *
+     * @return the name of the token.
+     * @throws IOException              if a connection problem with the RPC node arises.
+     * @throws ErrorResponseException   if the call to the node lead to an error. Not due to the
+     *                                  contract invocation itself but due to the call in general.
+     */
+    public String name() throws IOException, ErrorResponseException {
+        return this.testInvoke("name", null).getStack().get(0).asByteArray().getAsString();
+    }
+
+    /**
+     * Gets the total supply of the token.
+     *
+     * @return the total token supply deployed in the system.
+     * @throws IOException              if a connection problem with the RPC node arises.
+     * @throws ErrorResponseException   if the call to the node lead to an error. Not due to the
+     *                                  contract invocation itself but due to the call in general.
+     */
+    public BigInteger totalSupply() throws IOException, ErrorResponseException {
+        return this.testInvoke("totalSupply", null).getStack().get(0).asByteArray().getAsNumber();
+    }
+
+    /**
+     * Gets the symbol of the token.
+     *
+     * @return the symbol of the token.
+     * @throws IOException              if a connection problem with the RPC node arises.
+     * @throws ErrorResponseException   if the call to the node lead to an error. Not due to the
+     *                                  contract invocation itself but due to the call in general.
+     */
+    public String symbol() throws IOException, ErrorResponseException {
+        return this.testInvoke("symbol", null).getStack().get(0).asByteArray().getAsString();
```

Comment:
We need to make this a bit more robust.
What if the `.get(0)` doesn't exist? It will break everything.
Can't we use `.stream().map()` here, for example? Or, `.stream().findFirst()`?

Example:
```
        return abi.getMethods()
            .stream()
            .filter(f -> !isEmpty(f.getName()))
            .filter(f -> f.getName().equals(functionName))
            .findFirst()
            .map(NeoContractMethod::getParameters)
            .orElse(Collections.emptyList());
```

Lamdas and streaming are your friends. :-)

Please, also change other lines that use `.get(0)` and make one test for it. Imagine that, somehow, for whatever reason, the node returns something wrong or not expected by us in the stack (i.e., imagine that position 0 does not exist).